pkg/config/vars.go

```diff
@@ -147,6 +147,15 @@ func setKwokctlConfigurationDefaults(config *v1alpha1.KwokctlConfiguration) *v1a
 	conf.QuietPull = format.Ptr(envs.GetEnvWithPrefix("QUIET_PULL", *conf.QuietPull))
 
 	conf.Runtime = envs.GetEnvWithPrefix("RUNTIME", conf.Runtime)
+	if len(conf.Runtimes) == 0 {
+		conf.Runtimes = []string{
+			consts.RuntimeTypeDocker,
+			consts.RuntimeTypeNerdctl,
```

Comment:
#255 tried to add e2e for nerdctl under darwin, but it doesn't seem to work, so better to just add it for Linux?

Reply:
previously I was trying to add nerdctl for mac, because I thought it would be suitable for the runtime detection, but your idea using configure file seems easier, would use this way for runtime detection 

Will raise another PR for mac nerdctl, I am a mac nerdctl user, it should be easy to add it

Reply:
note: Yeah, Nerdctl as a secondary option, if e2e is not enabled under darwin before the next release, I plan to change it to be used only under Linux, just like the binary.